src/app/games/coerceo/CoerceoState.ts

```diff
@@ -91,7 +91,7 @@ export class CoerceoState extends TriangularGameState<FourStatePiece> {
     }
     public getCapturedNeighbors(coord: Coord): Coord[] {
         const OPPONENT: Player = this.getCurrentOpponent();
-        const neighbors: Coord[] = TriangularCheckerBoard.getNeighboors(coord);
+        const neighbors: Coord[] = TriangularCheckerBoard.getNeighbors(coord);
```

Comment:
tiens ligne 86 et 121, ce sont deux des fameux warning restant ("aliasing this to local variable"), une idée de comment s'en débarasser ? Si y'a pas moyen proprement, voir à rajouter un `// eslint-disable-next-line @typescript-eslint/no-this-alias` sur la ligne précédente